optimum/exporters/onnx/model_configs.py

```diff
@@ -111,6 +111,10 @@ def inputs(self) -> Mapping[str, Mapping[int, str]]:
         return {"input_ids": dynamic_axis, "attention_mask": dynamic_axis}
 
 
+class MPNetOnnxConfig(DistilBertOnnxConfig):
+    DEFAULT_ONNX_OPSET = 12
```

Comment:
Just for my knowledge, is opset 11 failing for this model? If you want, you can add a comment as to why opset 12 is required.

Reply:
Here the error when export with opset 11:
```
An error occured with the error message: [ONNXRuntimeError] : 10 : INVALID_GRAPH : Load model from here/model.onnx failed:This is an invalid model. Type Error: Type 'tensor(int64)' of input parameter (/encoder/Add_1_output_0) of operator (Min) in node (/encoder/Min) is invalid..
```
Apparently the Min operator is not compliant with int64 tensors (as far as I understand this message)